packages/cardano-services/src/StakePoolSearch/DbSyncStakePoolSearchProvider/queries.ts

```diff
@@ -10,47 +10,284 @@ export const findLastEpoch = `
  LIMIT 1
 `;
 
+export const findTotalAda = `
```

Comment:
we can use simpler query from the dbsync doc:
select sum (value)
from tx_out as tx_outer where
    not exists
        ( select tx_out.id
          from tx_out
          inner join tx_in on
              tx_out.tx_id = tx_in.tx_out_id and
              tx_out.index = tx_in.tx_out_index
          where tx_outer.id = tx_out.id
        );

Reply:
This query performs better than the current one. But Ive found that the results doesnt match. Will take a deeper look at it